CryptoSwift.xcodeproj/xcshareddata/xcschemes/CryptoSwift.xcscheme

```diff
@@ -22,10 +22,10 @@
          </BuildActionEntry>
          <BuildActionEntry
             buildForTesting = "YES"
-            buildForRunning = "YES"
-            buildForProfiling = "YES"
+            buildForRunning = "NO"
+            buildForProfiling = "NO"
             buildForArchiving = "NO"
-            buildForAnalyzing = "YES">
+            buildForAnalyzing = "NO">
```

Comment:
do we have to disable all?

Reply:
This is meant to be more of a hotfix...better to have 0.8.x working in the meantime